src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -1859,7 +1861,7 @@ void completeTermination() {
                 waiters = waiters.getNext();
             }
             tail.setNext(TERMINATE_COMPLETE);
-            if (! DISABLE_MBEAN) {
```

Comment:
native-image can no longer DCE this path, which can result in failures due to JMX now being reachable.

Reply:
ouch, sorry!